lasso/emme.py

```diff
@@ -132,6 +132,10 @@ def create_emme_network(
     )
 
     WranglerLogger.info("Make sure the CRS of input network is correct")
+
+    # problem: reprojection causes inf in lat/long when converting from WGS 4326
+    links_df.crs = CRS("epsg:4269")  
```

Comment:
Add condition here to first check if the input links_df and nodes_df are using WGS 84, if so, overwrite CRS to be NAD 83.